pennylane/devices/experimental/default_qubit_2.py

```diff
@@ -137,8 +138,8 @@ def name(self):
         """The name of the device."""
         return "default.qubit.2"
 
-    def __init__(self, seed="global", max_workers=None) -> None:
-        super().__init__()
+    def __init__(self, shots=None, seed="global", max_workers=None) -> None:
```

Comment:
do we have any interest in making a habit of adding `*, ` to force kwargs to be named? I'm asking because order _shouldn't_ matter, but the word "shouldn't" means very little in our little world.

Reply:
Potentially a good idea to consider for the future.